docs/LanguageGuide.md

```diff
@@ -227,6 +156,88 @@ These can only be used at the REPL and during macro evaluation. Here's a subset
 
 To see all functions available in the `Dynamic` module, enter `(info Dynamic)` at the REPL.
 
+
+### Structs
+Any structure type defined in Carp has an init method that can be used to create a new instance. It must be called with all the arguments in the order they are defined.
```

Comment:
Could we wrap `init` in backticks here?

Also, maybe "in the order in which they are defined" or "in definition order" might be better?